api/dp-openapi.yaml

```diff
@@ -1175,17 +1283,6 @@ components:
       schema:
         type: string
 
-  requestBodies:
```

Comment:
Can't we restore this?

Reply:
Do you mean just the heading 'requestBodies'?

Reply:
No, the whole `page_text`. You removed this schema object because you needed to change it in one instance, but we're no longer doing that -- you reverted that change -- so why can't we restore this schema object and its use in the two places?